numpy/lib/tests/test_shape_base.py

```diff
@@ -38,7 +38,7 @@ def test_argequivalent(self):
             (np.sort, np.argsort, dict()),
             (_add_keepdims(np.min), _add_keepdims(np.argmin), dict()),
             (_add_keepdims(np.max), _add_keepdims(np.argmax), dict()),
-            (np.partition, np.argpartition, dict(kth=2)),
+            #(np.partition, np.argpartition, dict(kth=2)),
```

Comment:
I wasn't sure what to do with this test. When there is no unique defined output to `np.partition` or `np.argpartition`, we obviously cannot expect their outputs to match with each other, right?

Reply:
> we obviously cannot expect their outputs to match with each other, right?

I agree, the order is supposed to be undefined.